### PR TITLE
Fixes #243 and #245 fixed position for scenegraph toolbar

### LINKED ISSUE #243: Back to Scene button overlaps the scroll bar (Firefox / Windows)
The close button overlaps the scroll bar of the scene panel. scroll

### PR DESCRIPTION
**Pull request by fernandojsg**
scroll

### CODE REVIEW

File: src/css/main.css [hunk 1]
```diff
@@ -938,6 +940,10 @@ code, pre {
   outline: none;
   flex: 1 1 auto;
   overflow-y: auto;
+  position: fixed;
+  width: 200px;
+  top: 98px;
+  height: calc(100% - (98px));
```
Comment (dmarcos) [inline src/css/main.css]: Extra set of parenthesis around 98px

Comment (fernandojsg) [inline src/css/main.css]: Yep, fixed!

**Comment by dmarcos** [comment]
one nit. it looks good to me

**Comment by fernandojsg** [comment]
thanks!

**Pull request merged**